pkgs/modules/default.nix

```
@@ -35,7 +35,14 @@ let
     })
     (import ./nodejs-with-prybar)
 
-    (import ./go)
+    (import ./go {
+      inherit (pkgs) go gopls;
+    })
+    (import ./go {
+      inherit (pkgs-unstable) gopls;
```

```suggestion
      gopls = pkgs-unstable.gopls.override { buildGoModule = pkgs.buildGo121Module; };
```
Let's make sure the gopls matches the same version as go.